### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by RajaRakoto**
- add preloader
- add swiper (carousel)
- set background style
- set typography
- init animations
- add normalize

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>HIU</title>
+		<link rel="stylesheet" href="./assets/main_style.css" />
+		<link rel="stylesheet" href="./assets/normalize.css" />
+		<link rel="stylesheet" href="./assets/countdown.css" />
+		<link rel="stylesheet" href="./assets/animations.css" />
+		<link rel="stylesheet" href="./assets/lib/swiper-bundle.min.css" />
+		<link rel="stylesheet" href="./assets/swiper.css" />
```
Comment (judicaelandria) [inline index.html:13]: ataovy preconnect ilay link

File: index.html [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+			</div>
+			<div id="countdown_container">
+				<div class="count_down_item" id="day_container"></div>
+				<div class="count_down_item" id="hour_container"></div>
+				<div class="count_down_item" id="min_container"></div>
+				<div class="count_down_item" id="sec_container"></div>
+			</div>
+		</div>
+
+		<script src="./assets/lib/swiper-bundle.min.js"></script>
```
Comment (judicaelandria) [inline index.html:44]: idem (maninona raha nampiasa lien CDN?

File: index.html (hunk 1 above)
Comment (judicaelandria) [reply index.html:13]: mampiasa cdn tsara kokoa

**Review by judicaelandria — CHANGES_REQUESTED** [review]

**Comment by judicaelandria** [comment]
Tsara ilay idea fa misy suggestion eo ambony @RajaRakoto

**Comment by julkwel** [comment]
Tsara ilay idée, à voir ny responsive. @judicaelandria  @RajaRakoto

**Pull request merged**